sdk/search/azure-search-documents/azure/search/documents/_generated/_serialization.py

```diff
@@ -1488,7 +1488,7 @@ def __call__(self, target_obj, response_data, content_type=None):
         :param str target_obj: Target data type to deserialize to.
         :param requests.Response response_data: REST response object.
         :param str content_type: Swagger "produces" if available.
-        :raises: DeserializationError if deserialization fails.
+        :raises SerializationError: if serialization fails.
```

Comment:
Looks like this will raise a DeserializationError, not SerializationError


https://github.com/Azure/azure-sdk-for-python/blob/7891da3a6fd12fd5508e7a6df495ea24090c1eef/sdk/search/azure-search-documents/azure/search/documents/_generated/_serialization.py#L1567